tools/buildsys/src/manifest.rs

```diff
@@ -220,6 +220,7 @@ struct Metadata {
 
 #[derive(Deserialize, Debug)]
 #[serde(rename_all = "kebab-case")]
+#[allow(dead_code)]
```

Comment:
What is the dead code?

Reply:
https://github.com/bottlerocket-os/bottlerocket/pull/1748 added release URLs to `BuildPackage`, but it isn't currently in use by any code. This allows the "dead code" only in the scope of  `BuildPackage`.